src/styles/styles.js

```diff
@@ -1389,9 +1420,9 @@ const webViewStyles = {
             ...baseCodeTagStyles,
             paddingLeft: 5,
             paddingRight: 5,
-            paddingBottom: 2,
-            alignSelf: 'flex-start',
             fontFamily: fontFamily.MONOSPACE,
+            lineHeight: 18,
+            fontSize: 13,
```

Comment:
The inline code block font size was being inherited. But after explicitly setting it to 13px, it's always rendered at 13px even if the parent is a `H1` where the font size is expected to be a little bigger. (Coming from https://github.com/Expensify/App/issues/30203)